scripts/docker/debian/sid/Dockerfile

```diff
@@ -71,11 +71,6 @@ RUN mkdir -p ${GTEST_ROOT} \
     && tar -zxvf gtest.tar.gz --strip-components=1 -C ${GTEST_ROOT} \
     && rm gtest.tar.gz
 
-# Handle Java
-RUN echo 'export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")'>> /etc/bash.bashrc
-RUN echo '\
-/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/\n\
-/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server/\n' > /etc/ld.so.conf.d/jdk.conf
 RUN ldconfig
```

Comment:
Is this needed?

Reply:
I do not think so. At least the build job `debian-unstable-full` shows the same amount of executed tests (249) [before](https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/master/562/pipeline/420) and [after](https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2568/5/pipeline/449/) the update.

Reply:
I was referring to "RUN ldconfig".

Reply:
> I was referring to "RUN ldconfig".

As far as I can tell, we do not need to run `ldconfig`, since none of the commands before it install a library manually.